generator.rb

```diff
@@ -0,0 +1,59 @@
+def is_vowel?(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    letter = true
+  else
+    letter = false
+  end
+  return letter
```

Comment:
I would probably not store the `true` or `false` value in `letter`, because we would expect `letter` to be a one-character string. Instead you could create a new variable (maybe `vowel`), or just return from inside the `case` statement.